Makefile

```diff
@@ -58,7 +58,7 @@ bin:
 
 .PHONY: build
 build: ## build binaries
-	$(DOCKER_CMD) go build $(GOGCFLAGS) -o bin/machine-controller-manager -ldflags '-extldflags "-static"' sigs.k8s.io/cluster-api-provider-aws/cmd/manager
+	$(DOCKER_CMD) go build $(GOGCFLAGS) -o bin/machine-controller-manager -ldflags '-X sigs.k8s.io/cluster-api-provider-aws/pkg/version.Raw=$(shell git describe --abbrev=8 --dirty --always) -extldflags "-static"' sigs.k8s.io/cluster-api-provider-aws/cmd/manager
```

Comment:
Wouldn't block the PR on this, but this is a little hard to read. Elsewhere we do this at the top of the Makefile:
```makefile
VERSION     ?= $(shell git describe --always --dirty --abbrev=7)
LD_FLAGS    ?= -X $(REPO_PATH)/pkg/version.Raw=$(VERSION)
BUILD_DEST  ?= bin/cluster-autoscaler-operator
```

Then in the build target:
```makefile
.PHONY: build
build: ## build binaries
        $(DOCKER_CMD) go build $(GOGCFLAGS) -ldflags "$(LD_FLAGS)" -o "$(BUILD_DEST)" "$(REPO_PATH)/cmd/manager"
```

Reply:
Thanks, will change it.